src/main/php/PHPMD/RuleSetFactory.php

```diff
@@ -109,8 +112,6 @@ public function createRuleSets(string $ruleSetFileNames): array
      * Creates a single rule-set instance for the given filename or identifier.
      *
      * @param string $ruleSetOrFileName The rule-set filename or identifier.
-     * @throws RuleSetNotFoundException
-     * @throws RuntimeException
      */
```

Comment:
Likely just lost in rebasing
```suggestion
     * @throws RuleSetNotFoundException
     * @throws RuntimeException
     */
```